### PR TITLE
Completed all the tasks

### PR DESCRIPTION
**Pull request by bharatkram**

### CODE REVIEW

**Comment by bharatkram** [comment]
For task 1, i created a table from the adminer webpage given.

File: project1/templates/admin.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                <th>Time</th>
+                <th>Type</th>
+            </tr>
+            {% for user in users %}
+                <tr>
+                    {% for attr in user %}
+                        <td>{{ attr }}</td>
+                    {% endfor %}
+                </tr>
+            {% endfor %}
```
Comment (madhulika9293) [inline project1/templates/admin.html:52]: nice job with nested for loops!

File: project1/application.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ def register():
     else:
         name = request.form.get("usr")
         user = db.execute("SELECT username FROM users WHERE username = :name", {"name": name}).fetchone()
+        print(user)
         if user is None:
             psw = request.form.get("psw")
             db.execute("INSERT INTO users (username, password, time) VALUES (:name, :psw, :time)", {"name":name, "psw":psw, "time":datetime.datetime.now()})
```
Comment (madhulika9293) [inline project1/application.py]: write orm queries if possible ie., in python. not SQL code.

**Comment by madhulika9293** [comment]
You can merge this to master.

File: project1/application.py (hunk 2 above)
Comment (bharatkram) [reply project1/application.py]: i will change them today after today's tasks.

**Pull request merged**